build/zlib-1.2.8/contrib/vstudio/vc14/zlibstat.vcxproj

```diff
@@ -171,38 +171,11 @@
     <CodeAnalysisRules Condition="'$(Configuration)|$(Platform)'=='Release|x64'" />
     <CodeAnalysisRuleAssemblies Condition="'$(Configuration)|$(Platform)'=='Release|x64'" />
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='ReleaseWithoutAsm|Win32'">
```

Comment:
I may have created that vc14 folder, I don't remember. I would have started with a copy of vc11 and just made whatever changes applied. So if it's not in the official source, it doesn't much matter which you do, but it's probably best to stick to what is in the official source as much as possible and only change what needs to be changed.